test/regression.uts

```diff
@@ -19,6 +19,23 @@ ls()
 ~ conf command
 lsc()
 
+= List contribs
+import mock
+result_list_contrib = ""
+def test_list_contrib():
+    def write(s):
+        global result_list_contrib
+        result_list_contrib += s
+    mock_stdout = mock.Mock()
+    mock_stdout.write = write
+    sys.stdout = mock_stdout
+    list_contrib()
+    sys.stdout = sys.__stdout__
```

Comment:
Please never use this !
That breaks the logs system !

sys.stdout is overwritten with the autorun utils, if you want to overwrite it in your tests (I guess they are failing because of that), never use `sys.__stdout__` as the `autorun` value will be lost.

Use a tmp value to store it